### ISSUE TITLE
UselessInheritDocComment false negative

### ISSUE THREAD

**Issue by mabar** [slevomat/coding-standard#1411]
From following cases, only last one is reported by `SlevomatCodingStandard.Commenting.UselessInheritDocComment`. I believe even parameters and returns with no explicit type should follow inheritance rules and should be reported as useless

```php
<?php

/**
 * {@inheritDoc}
 */
function a(mixed $a)
{
}

/**
 * {@inheritDoc}
 */
function b($a): void
{
}

/**
 * {@inheritDoc}
 */
function c($a)
{
}

/**
 * {@inheritDoc}
 */
function d(mixed $a): void
{
}


```

**Comment by kukulich** [comment]
See https://github.com/slevomat/coding-standard/issues/441

I agree with you but the `@inheritdoc` annotation is also used to block other reports.

**Comment by mabar** [comment]
Could it be configurable please? I've already disabled these sniffs that are suppressed by inheritdoc, because PHPStan does the same checks.

Namely `SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingAnyTypeHint` and `SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingAnyTypeHint` have better alternatives in PHPStan

**Comment by kukulich** [comment]
@mabar Yes, it could be configurable.

**Comment by mabar** [comment]
It would be helpful, thank you for considering it 🙏
